models/org.go

```diff
@@ -22,12 +24,26 @@ var (
 
 // IsOwnedBy returns true if given user is in the owner team.
 func (org *User) IsOwnedBy(uid int64) bool {
-	return IsOrganizationOwner(org.ID, uid)
+	isOwner, err := IsOrganizationOwner(org.ID, uid)
+	if err != nil {
+		log.Error(4, "IsOwnedBy: %v", err)
```

Comment:
And I don't know if the log will affect the SSH output? Maybe change the function's signature to `IsOwnedBy(uid int64) (bool, error)` or maybe just ignore them.

Reply:
Oh, I had thought this function was used in templates (and thus did not want to change the signature), but it looks like it isn't. Will update signature.

Reply:
Also, I don't see how this specific call to `log.Error` is different than any call to `log.Error` currently in the DB, and therefore I don't see why this particular `log.Error` raises concerns about SSH output that other `log.Error` calls do not.